test/syscalls/linux/pty.cc

```diff
@@ -1244,6 +1244,33 @@ TEST_F(PtyTest, SetMasterWindowSize) {
   EXPECT_EQ(retrieved_ws.ws_col, kCols);
 }
 
+TEST_F(PtyTest, DiscardPendingInputAndSet) {
```

Comment:
I think it's worth copying this test and switching master and slave.

Reply:
added, but actually as i currently understand, this test for master doesn't make much sense in gvisor context because we don't have output to drain.